frontend/src/routes/(app)/project/[owner]/[project]/(metadata)/+layout.svelte

```diff
@@ -1,5 +1,8 @@
 <script lang="ts">
 	import MetadataPanel from '$lib/components/metadata/MetadataPanel.svelte';
+	import { decodeURLParameters } from '$lib/util/util';
+
+	decodeURLParameters();
```

Comment:
Should this happen on a higher level? What if we have params outside a project? We probably will in the future with reports.

Reply:
hmm good point